tesseract_motion_planners/src/core/utils.cpp

```diff
@@ -40,6 +40,150 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 namespace tesseract_planning
 {
+tesseract_common::Toolpath toToolpath(const Instruction& instruction,
+                                      const tesseract_environment::Environment::ConstPtr& env)
+{
+  using namespace tesseract_planning;
+
+  if (env == nullptr)
+    throw std::runtime_error("toToolpath: The environment is a nullptr!");
+
+  tesseract_common::Toolpath toolpath;
```

Comment:
Out of curiosity, does a vector containing a vector of Eigen objects also need the special aligned allocator? I was under the impression that only the inner vector needs it

Reply:
Yea anything that contains an Eigen aligned object must also use the aligned operator. Same reason classes that include member objects that require the Eigen alignment then you also have to have it in your class. It is infections.